src/styles/components/tile.scss

```diff
@@ -37,14 +37,14 @@ $size: 15rem;
     bottom: 0.1em;
   }
 
-  &.tile--small {
+  &--small {
     height: $size/4;
     .tile__header {
       font-size: 1.2rem;
     }
   }
 
-  &.tile--uniref {
+  &--uniref {
```

Comment:
Since the `namespace` is one of a limited set in the scss file we can also set the proptypes in Tile.jsx:
```PropTypes.oneOf(['uniref', 'uniparc', 'proteomes'])```